components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/dao/SAMLSSOPersistenceManagerFactory.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.core.dao;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+
+/**
+ * Factory class to create instances of SAMLSSOServiceProviderDAO based on the configured storage type.
+ */
+public class SAMLSSOPersistenceManagerFactory {
```

Comment:
Lets rename to `SAMLServiceProviderPersistenceManagerFactory`

Reply:
Resolved in [Introduce SAMLSSOServiceProviderDAO Factory class](https://github.com/wso2/carbon-identity-framework/pull/6240/commits/2c347ada3cf2d0754035586e395ccb929e51f1f4)